src/api/apiServices.js

```diff
@@ -33,7 +39,7 @@ export const get = async (url, params = {}) => {
     headers: getAuthHeaders(),
   });
 
-  return handleResponse(response);
+  return response.json();
```

Comment:
the return response should be uniform for each method. Like on above fucntion you are returning key value pair. So return here as well.